src/components/Button/Button.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import {TouchableOpacity, Text, ActivityIndicator} from 'react-native';
+import PropTypes from 'prop-types';
+import styles from '../../styles/styles';
+import themeColors from '../../styles/themes/default';
+
+const propTypes = {
+    /**
+     * Callback fired when the button is pressed.
+     */
+    onPress: PropTypes.func,
+
+    /**
+     * Size to display the button at
+     */
+    size: PropTypes.oneOf(['small', 'large']),
```

Comment:
Instead of large, I think it's actually just the default button size. So could we do `small` and `default` here?